api/spec/api_docs/config.rb

```diff
@@ -10,7 +10,7 @@ def swagger_root
 
       def definitions
         defined = {}
-        ApiDocs::Definition::Resource.constants.each do |resource_class|
+        ApiDocs::Definitions::Resources.constants.each do |resource_class|
```

Comment:
I can't understand why this change is happening. Is this even an actual class?

Reply:
I changed the name of some of the modules to match the folders in which they were located (eg. there was a `Definition` module located in the `defintions` folder). I was under the impression that this was following the rails convention. Is that correct?

Reply:
Yes, you have it right—I was confused about the module/class structure. Thanks for checking.